LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceClient.cs

```diff
@@ -143,7 +143,7 @@ public async Task<bool> SendEventAsync(LoRaDeviceTelemetry telemetry, Dictionary
 
                     return true;
                 }
-                catch (OperationCanceledException ex) when (ExceptionFilterUtility.True(() => this.logger.LogError($"could not send message to IoTHub/Edge with error: {ex.Message}")))
+                catch (OperationCanceledException) when (ExceptionFilterUtility.True(() => this.logger.LogError("could not send message to IoTHub/Edge due to timeout.")))
```

Comment:
pls always pass the original exception to the logger. The stack traces where the timeout happened can be crucial.

Reply:
Done in 6fcc890f3a924576e042a9b5fc4c635d4bed2343 (as well as other occurrences in the same file)